src/main/java/harmonised/pmmo/features/mobscaling/MobAttributeHandler.java

```diff
@@ -77,6 +79,7 @@ private static void handle(LivingEntity entity, ServerLevel level, Vec3 spawnPos
 
 		var dimMods = dimData.mobModifiers().getOrDefault(RegistryUtil.getId(entity), new HashMap<>());
 		var bioMods = bioData.mobModifiers().getOrDefault(RegistryUtil.getId(entity), new HashMap<>());
+		var dimModsCustom = dimData.dimensionalMobModifiers();
```

Comment:
Make sure you capture global modifiers from both biomes and dimensions.  I recommend you add a method in `LocationData` which returns the list of modifiers from both explicit and global sources.  eg
```java
public List<MobModifier> getModifiers(ResourceLocation mobID) {
  List<MobModifier> modifiers = new ArrayList<>(globalMobModifiers());
  if (mobModifiers().get(mobID) != null) modifiers.addAll(mobModifiers.get(mobID));
  return modifiers;
}
```
this of course requires implementation of my other suggestion to update the existing location mob modifier format.

Reply:
Had to do a refactor for the mob modifier in multiple places, please take a look, as I am not accustomed with the PMMO compats